Documentation for .innerText on <input/> is wrong

### Are you requesting a feature or reporting a bug?

Reporting a bug

### What is the current behavior?

`<input/>` values cannot be matched using `.innerText` but docs suggest that they can.

### What is the expected behavior?

Either the documentation should be updated, or `.innerText` should match the value of an input field.

Also, having an example of getting an input's value would be useful, for example:
```js
import { Selector } from 'testcafe';

fixture `My fixture`
    .page `http://devexpress.github.io/testcafe/example/`;

test('Assertion with Selector', async t => {
    const developerNameInput = Selector('#developer-name');

    await t
        .expect(developerNameInput.value).eql('')
        .typeText(developerNameInput, 'Peter')
        .expect(developerNameInput.value).eql('Peter');
});
```

### How would you reproduce the current behavior (if this is a bug)?
Run the test from the docs on a page with `<input id="developer-name"/>`. According to the [docs](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/using-selectors.html):
```js
import { Selector } from 'testcafe';

fixture `My fixture`
    .page `http://devexpress.github.io/testcafe/example/`;

test('Assertion with Selector', async t => {
    const developerNameInput = Selector('#developer-name');

    await t
        .expect(developerNameInput.innerText).eql('')
        .typeText(developerNameInput, 'Peter')
        .expect(developerNameInput.innerText).eql('Peter');
});
```
In fact, this test fails with `AssertionError: expected '' to deeply equal 'Peter'`


### Specify your

* operating system: OSX
* testcafe version: 0.21.1
* node.js version: v9.11.2

--- a/docs/articles/documentation/test-api/selecting-page-elements/selectors/using-selectors.md
+++ b/docs/articles/documentation/test-api/selecting-page-elements/selectors/using-selectors.md
@@ -162,9 +162,9 @@ test('Assertion with Selector', async t => {
     const developerNameInput = Selector('#developer-name');
 
     await t
-        .expect(developerNameInput.innerText).eql('')
+        .expect(developerNameInput.value).eql('')
         .typeText(developerNameInput, 'Peter')
-        .expect(developerNameInput.innerText).eql('Peter');
+        .expect(developerNameInput.value).eql('Peter');
 });
 ```
 

